custom/crediblex/portfolio/loanaccount/src/main/java/com/crediblex/fineract/portfolio/loanaccount/service/CustomLoanProductReadPlatformServiceImpl.java

```diff
@@ -129,16 +94,27 @@ public String loanProductSchema() {
                     + " left join m_floating_rates as fr on lfr.floating_rates_id = fr.id "
                     + " left join m_product_loan_variable_installment_config as lvi on lvi.loan_product_id = lp.id "
                     + " left join m_delinquency_bucket as dbuc on dbuc.id = lp.delinquency_bucket_id "
+                    + " left join m_line_of_credit loc on loc.id = l.line_of_credit_id "
```

Comment:
What is l in this case? I dont see any entity l in the query.

Reply:
L was meant for loan, i removed the loan join but forgot to clean up this join as well